src/backend/OpenCL/CHIPBackendOpenCL.cc

```diff
@@ -470,19 +470,23 @@ CHIPKernelOpenCL::CHIPKernelOpenCL(const cl::Kernel &&ClKernel,
 // CHIPContextOpenCL
 //*************************************************************************
 
-void CHIPContextOpenCL::freeImpl(void *Ptr) { UNIMPLEMENTED(); };
+void CHIPContextOpenCL::freeImpl(void *Ptr) {
+  size_t Ignored;
+  SvmMemory.free(Ptr, &Ignored);
```

Comment:
What is the purpose of the size argument?

Reply:
@linehill

Reply:
No idea what the original author had in mind for the argument. Seems unnecessary so I removed it.